src/events/xwayland_events.rs

```diff
@@ -84,7 +85,13 @@ impl ResizeEvent {
     }
 
     /// Get the resize edge information for the resize action.
-    pub fn edges(&self) -> uint32_t {
-        unsafe { (*self.event).edges }
+    pub fn edges(&self) -> Edges {
+        unsafe {
+            let edges_bits = (*self.event).edges;
+            match Edges::from_bits(edges_bits) {
```

Comment:
Just use `expect`

Reply:
also just use `expect` in those other cases obviously.

Reply:
oh wait then you can't put in `edge_bits` (tbh I didn't realize you could do that with `panic` and not `expect`. I don't `panic` explicitly often).

eh whatever